com.github.wjaguar.mtpaint.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: com.github.wjaguar.mtpaint
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: mtpaint
+rename-icon: mtpaint
+rename-desktop-file: mtpaint.desktop
+rename-appdata-file: mtpaint.appdata.xml
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Network access
+  # for printing
+  - --share=network
+  # Filesystem access
+  # xdg-desktop-portal-gtk does not support Gtk+2
+  # even if, this program requires direct access to files
+  - --filesystem=host
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - openjpeg.yaml
+  - imagemagick.yaml
+  - yad.yaml
+  - gifsicle.yaml
+  - exif.yaml
+  - leafpad.yaml
+  - vte.yaml
+  - lxterminal.yaml
+  - mtpaint.yaml
```

Comment:
Why does painting program require text editor and terminal emulator?

Reply:
> Why does painting program require text editor and terminal emulator?

See my comment above.
https://github.com/flathub/flathub/pull/745#discussion_r238206983